client/task-list/tasks/payments/PaymentGatewaySuggestions/components/Setup/Connect.js

```diff
@@ -94,22 +95,24 @@ export const Connect = ( {
 	const helpText = setupHelpText && (
 		<p dangerouslySetInnerHTML={ sanitizeHTML( setupHelpText ) } />
 	);
-	const DefaultForm = ( props ) => (
-		<DynamicForm
-			fields={ fields }
-			isBusy={ isUpdating }
-			onSubmit={ handleSubmit }
-			submitLabel={ __( 'Proceed', 'woocommerce-admin' ) }
-			validate={ ( values ) => validateFields( values, fields ) }
-			{ ...props }
-		/>
-	);
+	const DefaultForm = useRef( ( props ) => {
```

Comment:
`useRef` works here to prevent the re-render, but also prevents props from working here (busy state will no longer work and if fields were updated, they would not work).

Reply:
Yeah I do agree, `useRef` felled like a dirty fix (but did the trick). I think I will refactor this a bit.

Reply:
Fixed here, I end up just using the rendered element: https://github.com/woocommerce/woocommerce-admin/pull/7256/commits/f9d5b0a1173aca18a7aea21857c889786724ff8e
This can still be used by using `{ defaultForm }` or `cloneElement(defaultForm, newProps)`.